src/main/osd/fc_state.h

```diff
@@ -62,6 +62,9 @@ typedef struct fcStatus_s {
     uint16_t amperage;                 // amperage in 0.01A steps, 12575 = 125.75A
     uint16_t mAhDrawn;                 // milliampere hours, 1300mAh
 
+    // MSP_ALTITUDE
+    uint16_t altitudeBaroM;            // altitude as reported by the baro (or sonar), in meters
```

Comment:
I think 16 bits should be enough - if someone is flying their quad to outer space, let me know and I'll expand this. :)